test/fixtures/experimental/class-private-properties/asi-failure-computed/actual.js

```diff
@@ -1,4 +1,4 @@
 class Foo {
   #p = x
-  [#m] () {}
```

Comment:
Upps, the #4 was not intended to be there. Basically I wanted to show initializing a private field from another private filed.